ci/setup_env.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash -e
+
+
+# edit the locale file if needed
+function edit_init()
+{
+    if [ -n "$LOCALE_OVERRIDE" ]; then
+        echo "[Adding locale to the first line of pandas/__init__.py]"
```

Comment:
I think we removed the squared brackets from everywhere except this block. Can you remove them from here too please?